src/Compilers/CSharp/Portable/Binder/NameofBinder.cs

```diff
@@ -49,14 +51,42 @@ private bool IsNameofOperator
 
         internal override void LookupSymbolsInSingleBinder(LookupResult result, string name, int arity, ConsList<TypeSymbol> basesBeingResolved, LookupOptions options, Binder originalBinder, bool diagnose, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
         {
+            bool foundParameter = false;
+            if (_withParametersBinder is not null && IsNameofOperator)
+            {
+                _withParametersBinder.LookupSymbolsInSingleBinder(result, name, arity, basesBeingResolved, options, originalBinder, diagnose, ref useSiteInfo);
+                if (!result.IsClear)
+                {
+                    if (result.IsMultiViable)
+                    {
+                        return;
```

Comment:
>return; [](http://example.com/codeflow?start=24&length=7)

Do we have a test confirming that parameters win over type parameters?

Reply:
Added